samples/cubes/convertStatusToString.js

```diff
@@ -1,96 +1,78 @@
+let EnumDSSCubeStates = require('./EnumDSSCubeStates');
+
 module.exports = (status) => {
-  const EnumDSSCubeStates = {
-    DssCubeReserved : 0,
-    DssCubeProcessing : 1,
-    DssCubeActive : 2,
-    DssCubePersisted : 4,
-    DssCubeInfoDirty : 8,
-    DssCubeDirty : 16,
-    DssCubeLoaded : 32,
-    DssCubeReady : 64,
-    DssCubeLoadPending : 128,
-    DssCubeUnloadPending : 256,
-    DssCubePendingForEngine : 512,
-    DssCubeImported : 1024,
-    DssCubeForeign : 2048,
-    DssCubeRemapDirty : 4096,
-    DssCubeUpgrade : 8192,
-    DssCubeEmma : 16384,
-    DssCubeFromMSTRFile : 32768,
-    DssCubeHidden : 65536
-  }
   let results = {};
-  if(status>=EnumDSSCubeStates.DssCubeHidden){
+  if(status>=EnumDSSCubeStates().DssCubeHidden){
```

Comment:
space please between `if` and condition:
```
  if (status>=EnumDSSCubeStates().DssCubeHidden){
```

I'll add something soon to make this formatting more automatic.